src/Nox.ClientApi.Tests/Generated/Nox.Generator/Nox.Generator.NoxCodeGenerator/Presentation.Api.OData.CountriesController.g.cs

```diff
@@ -91,9 +92,17 @@ public async Task<ActionResult> PostToCountryLocalNames([FromRoute] System.Int64
             return NotFound();
         }
         
-        return Created(new CountryLocalNameDto { Id = createdKey.keyId });
+        var child = await TryGetCountryLocalName(key, createdKey);
+        if (child == null)
+        {
+            return NotFound();
+        }
+        
+        return Created(child);
     }
     
+    [ODataIgnored]
+    [HttpPut("/api/Countries/{key}/CountryLocalNames/{relatedKey}")]
```

Comment:
I tested this on our test app, I think we could do it like this, it will be available in Odata and Swagger:

```
  [HttpPut("api/Tenants/{key}/OwnerContacts/{relatedKey}")]        
  public async Task<ActionResult> PutToOwnerContactsNonOdataConvention([FromODataUri] int key, [FromODataUri] int relatedKey, [FromBody] OwnerContact contactPersonDetails)
  {
      return NotFound();
  }
```

![image](https://github.com/NoxOrg/Nox.Generator/assets/8514193/061653df-faeb-45c1-b7f1-1307ac15467c)